dockereve-master/eve-app/settings.py

```diff
@@ -104,14 +204,109 @@
                 'TotalScanTimeSec': {'type': 'integer'},
                 'TransmitCoilName': {'type': 'string'},
                 'VariableFlipAngleFlag': {'type': 'string'},
-                # required data 
+
+                'ContrastBolusIngredient':  {'type': 'string'},
+                'Manufacturer': {'type': 'string'},
+                'HardcopyDeviceSoftwareVersion': {'type': 'string'},
+                'GradientSetType': {'type': 'string'},
+                'MRTransmitCoilSequence':{'type': 'string'},
+                'MatrixCoilMode': {'type': 'string'},
+                'CoilCombinationMethod': {'type': 'string'},
+                'PulseSequenceType': {'type': 'string'},
+                'PulseSequenceDetails': {'type': 'string'},
+                'NumberShots' :{'type': 'integer'},
+                'ParallelReductionFactorInPlane':{'type': 'float'},
+                'ParallelAcquisitionTechnique': {'type': 'string'},
+                'PartialFourier' :{'type': 'boolean'},
+                'PartialFourierDirection':{'type': 'string'},
+                'EffectiveEchoSpacing' :{'type': 'float'},
+                'TotalReadoutTime':{'type': 'float'},
+                'SliceEncodingDirection': {'type': 'string'},
+                'NumberOfVolumesDiscardedByScanner':{'type': 'float'},
+                'NumberOfVolumesDiscardedByUser':{'type': 'float'},
+                'DelayTime':{'type': 'float'},
+                'MultibandAccelerationFactor':{'type': 'float'},
+                'Instructions': {'type': 'string'},
+                'TaskDescription': {'type': 'string'},
+                'CogAtlasID' : {'type': 'string'},
+                'CogPOID': {'type': 'string'},
+                'InstitutionName': {'type': 'string'},
+                'InstitutionAddress': {'type': 'string'},
+                'ConversionSoftware': {'type': 'string'},
+                'ConversionSoftwareVersion': {'type': 'string'},
+            } #end of schema
+        },#end of bold
+          'T1w': {
+            'item_title': 'T1w-',
```

Comment:
Good point ; just cleaned up and pushed the updated version to my branch again. Should make another PR or wait until next time together.